workers/Dockerfile

```diff
@@ -1,5 +1,10 @@
+# based on Debian 11 Bullseye
 FROM python:3.8
 
+# install R
+RUN apt-get update
```

Comment:
Please don't include the apt cache in image layers.

Reply:
Thanks for pointing out. I will make sure not to include the apt-cache in the image layer by removing it after installation.

Reply:
Thanks for pointing out. I will make sure not to include the apt-cache in the image layer by removing it after installation.